ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -498,8 +498,8 @@ def install_replica(master, replica, setup_ca=True, setup_dns=False,
     return result
 
 
-def install_client(master, client, extra_args=(),
-                   user=None, password=None):
+def install_client(master, client, extra_args=(), user=None,
```

Comment:
`extra_args` default to tuple, however, they are only used upon conversion to list as seen at `531`. Thus, this conversion is obsolete and we could benefit from using .extend list method which would add list and tuple likewise.
```suggestion
def install_client(master, client, extra_args=[], user=None,
```

Reply:
Done